main/git/APKBUILD

```diff
@@ -159,21 +153,26 @@ p4() {
 daemon() {
 	pkgdesc="Git protocol daemon"
 	depends="git=$pkgver-r$pkgrel"
-	replaces="git"
 
 	mkdir -p "$subpkgdir"/$_gitcoredir
 	mv "$pkgdir"/$_gitcoredir/git-daemon \
 		"$pkgdir"/$_gitcoredir/git-http-backend \
 		"$pkgdir"/$_gitcoredir/git-shell \
 		"$subpkgdir"/$_gitcoredir/
+}
+
+daemon_openrc() {
+	pkgdesc="Git protocol daemon"
+	depends="git-daemon=${pkgver}-r${pkgrel} ${depends_openrc}"
```

Comment:
You should move depends below the call to default_openrc so that it is not reset by that function.

Reply:
ah yes, missed that one!